### DISCUSSION TITLE
vim: Persist macros across Zed sessions

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#23937] [Feature Requests]
### Check for existing issues

- [x] Completed

### Describe the bug / provide steps to reproduce it

Recording any macro in zed with`q<register>` and invoking the recorded macro with `@<register>` works as expected.

The only problem is that if I close zed and reopen it zed forgets about the macro completely.
I expect zed to remember macros I record between work sessions.

### Environment

archlinux
hyprland
zed 0.153.6-1 from official arch repositories (extra)

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by MaxThom** [comment]
+1 !

**Comment by Chuong98CC** [comment]
10 votes from me
